modules/combined/tests/elastic_thermal_patch/elastic_thermal_jacobian_rz_smp_test_sm.i

```diff
@@ -0,0 +1,220 @@
+# This problem is intended to exercise the Jacobian for coupled RZ
+# problems.  Only two iterations should be needed.
+
+[Problem]
+  coord_type = RZ
+[]
+
+[Mesh]
+  file = elastic_thermal_patch_rz_test.e
+[]
+
+[Functions]
+  [./ur]
+    type = ParsedFunction
+    value = '0'
+  [../]
+  [./uz]
+    type = ParsedFunction
+    value = '0'
+  [../]
+  [./body]
+    type = ParsedFunction
+    value = '-400/x'
+  [../]
+  [./temp]
+    type = ParsedFunction
+    value = '117.56+100*t'
+  [../]
+[]
+
+[Variables]
+  [./disp_x]
+    order = FIRST
+    family = LAGRANGE
```

Comment:
If it's not too much trouble, we may as well clean both versions up and keep them as consistent with each other as we can, even though the sm ones will be going away.